src/SablierMerkleLL.sol

```diff
@@ -112,6 +112,11 @@ contract SablierMerkleLL is
             timestamps.end = timestamps.start + _schedule.totalDuration;
         }
 
+        // Calculate the unlock amounts based on the percentages.
+        LockupLinear.UnlockAmounts memory unlockAmounts;
+        unlockAmounts.start = ud60x18(amount).mul(_schedule.startPercentage.intoUD60x18()).intoUint128();
+        unlockAmounts.cliff = ud60x18(amount).mul(_schedule.cliffPercentage.intoUD60x18()).intoUint128();
```

Comment:
since we can have the pct as 0, we could add `if(pct > 0) then mul`

but i decided not to do add this as it is not increasing the gas that much (0 mul is veerry cheap operation)
and it is shorter and clearer this one (just two lines ✨)

Reply:
ternary operator ftw? :))

Reply:
yeah, completely agree 😂